rentlinx.gemspec

```diff
@@ -12,4 +12,5 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'httpclient', '~> 2.6.0'
   s.add_runtime_dependency 'json', '~> 1.8'
+  s.add_runtime_dependency 'phony'
```

Comment:
That breaks compatibility with APM, unless we update Rentlinx every time we bump the version of Phony included in APM.